client/my-sites/plans-grid/components/plan-features-container.tsx

```diff
@@ -68,6 +133,9 @@ const PlanFeaturesContainer: React.FC< {
 						setActiveTooltipId={ setActiveTooltipId }
 						activeTooltipId={ activeTooltipId }
 					/>
+					{ config.isEnabled( 'bluehost-in-plans-grid' ) && isAgencyBlueHostPlan( planSlug ) && (
```

Comment:
Let's try not to use calypso-config package inside plans-grid. It's not portable outside. 

We mentioned this before I think?